docs/02-admin/01-installation/02-docker.md

```diff
@@ -202,6 +202,20 @@ In this example, port `8080` will connect to your Mbin server.
 > [!TIP]
 > In order to verify your server is correctly detecting it's public protocol (HTTP vs HTTPS), visit `/.well-known/nodeinfo` and look at which protocol is being used in the `href` fields. A public server should always be using HTTPS and not contain port numbers (i.e., `https://DOMAINHERE/`).
 
+### Additional configuration (Optional)
+
+If you run a larger Mbin instance, its recommended to increase the `shm_size` value of the `postgres` service (first try the default `2gb`!). Although you can also decrease the number if you wish on smaller instances. `shm_size` sets the size of the shared memory (`/dev/shm`) and used for dynamic memory allocation. PostgreSQL is using this for buffering the write-ahead logs (also known as "WALL buffer").
+
+The following step is **optional** and also depends on how much RAM you have left as well as how many parallel workers, table sizes, expected concurrent users and more. You can first use the default `2gb`, which should be sufficient, however below is explained how to further increase this number.
+
+1. In `compose.override.yaml`, add `shm_size` to the `postgres` service (`4gb` is an example here):
```

Comment:
```suggestion
In `compose.override.yaml`, add `shm_size` to the `postgres` service (`4gb` is an example here):
```

If there's not more than one "step", then it doesn't make much sense to use a list.

Reply:
Yes.. for now..... 🥹